torch/distributions/transforms.py

```diff
@@ -418,10 +418,6 @@ class TanhTransform(Transform):
     bijective = True
     sign = +1
 
-    @staticmethod
-    def atanh(x):
-        return 0.5 * (x.log1p() - (-x).log1p())
```

Comment:
I am concerned that the code looks like this for numeric stability reasons, but there is no comment to that effect here

Reply:
That's right. Specially for values close to 0, this will be more accurate than the `0.5*log((1+x)/(1-x))` formulation.